### PR TITLE
when 1.1.40

### PR DESCRIPTION
**Pull request by EricFromCanada**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/when.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class When < Formula
   desc "Tiny personal calendar"
   homepage "http://www.lightandmatter.com/when/when.html"
-  url "https://deb.debian.org/debian/pool/main/w/when/when_1.1.38.orig.tar.gz"
```
Comment (chenrui333) [inline Formula/when.rb:4]: saw this as well, looks like this is only on the github side yet. Not sure if we want to wait a little bit.

Comment (chenrui333) [reply Formula/when.rb:4]: `https://repology.org/project/when/versions` repology reference

**Comment by EricFromCanada** [comment]
The home page has been updated to refer to v1.1.40, so I'd say this is good to go.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Nice!

**Pull request closed without merging**